docs/dev_guide/ceph_s3_tests_guide.md

```diff
@@ -0,0 +1,138 @@
+In the core repository run the following to build noobaa core image with the desired tag to build the tester image:
+```bash
+make tester TESTER_TAG=noobaa-tester:s3-tests
+docker tag noobaa:latest noobaa-core:s3-tests
+```
+
+### 4) Deploy Noobaa (Noobaa-Operator)
+From the operator repository (where we have the alias to `nb`):
+```bash
+nb install --mini --noobaa-image='noobaa-core:s3-tests'
+```
+The installation should take 5-10 minutes.
+
+Once noobaa is installed, you should see something similar to this when getting the pods:
+```
+> kubectl get pods
+NAME                                               READY   STATUS    RESTARTS   AGE
+noobaa-core-0                                      1/1     Running   0          51m
+noobaa-db-pg-0                                     1/1     Running   0          51m
+noobaa-default-backing-store-noobaa-pod-a586c55b   1/1     Running   0          47m
+noobaa-endpoint-6cf5cccfc6-rmdrd                   1/1     Running   0          47m
+noobaa-operator-5c959d5564-qzgqb                   1/1     Running   0          51m
+```
+
+### 5) Wait For Default Backingstore to Be Ready
+```bash
+kubectl wait --for=condition=available backingstore/noobaa-default-backing-store --timeout=6m
+```
+
+## Run a Single Ceph S3 Test
+
+### 1) Requisitions:
+Following the General Settings For Ceph S3 Tests.
+
+### 2) Increasing Debug Level (Noobaa-Operator)
+Before running a test, you can increase the debug level with noobaa CLI.
+```bash
+nb system set-debug-level <level>
+```
+A good level to start with is 2 or 3. the higher you go the more verbose and noisy the logs will become.
+
+### 3) Deploy The Tester Deployment 
+Use the `test_ceph_s3_deployment.yml` file to install the tester pod. This will install a pod that does nothing, but it allows us to go into the pod and run our test.
+```bash
+kubectl apply -f src/test/system_tests/ceph_s3_tests/test_ceph_s3_deployment.yml
+```
+
+### 4) Setup Test Config (Inside The Tester Pod)
+Once the tester pod is up, we can go into it and prepare the environment to run the tests.
+both `kubectl` and `oc` can be used:
+```bash
+# if you have kubectl 
+kubectl exec -it [noobaa-tester pod] -- bash
+# or with oc
+oc rsh [noobaa-tester pod] bash
+```
+
+In the tester pod, go to noobaa working directory:
+```bash
+cd /root/node_modules/noobaa-core/
+```
+
+Run the script that will create the necessary accounts in noobaa and update the Ceph S3 tests config file accordingly:
+```bash
+node ./src/test/system_tests/ceph_s3_tests/test_ceph_s3_config_setup.js
+```
+
+Note: If you want to ignore PythonDeprecationWarnings use (which will then ignore all Python warnings, so keep that in mind):
+```bash
+export PYTHONWARNINGS="ignore"
+```
+
+### 5) Run a Test (Inside The Tester Pod)
+To run a test, from noobaa working directory:
+```bash
+S3TEST_CONF=src/test/system_tests/ceph_s3_tests/test_ceph_s3_config.conf ./src/test/system_tests/ceph_s3_tests/s3-tests/virtualenv/bin/nosetests <test_name>
```

Comment:
There are (at least) two formats for tests name. This needs an example to clarify expected format.

Reply:
You're right, I have a gap in the documentation... the full list that @utkarsh-pro wrote is not published because is in our drive.
@utkarsh-pro, any suggestion on how can I add the full list?

@alphaprinz, do you mean the difference of the test name between the file and the output after running it (`:` vs `.`)?

Reply:
@shirady, I am not sure if I get what is the ask here. Are we trying to figure out how to embed the list of all of the tests in this documentation?

Reply:
@shirady Yes. Also I think sometimes tests are referred by short name and sometime fully qualified name.

Reply:
I added a section for 'test name' and an example.